app/code/Bulbulatory/Recommendations/etc/db_schema.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+    <table name="bulbulatory_recommendations" engine="innodb" comment="Recommendations table">
+        <column xsi:type="int" name="id" padding="10" unsigned="true" nullable="false" identity="true"
+                comment="Recommendation ID"/>
+        <column xsi:type="int" name="customer_id" padding="10" unsigned="true" nullable="false"
+                comment="Customer entity id"/>
+        <column xsi:type="varchar" name="email" nullable="false" length="255" comment="Recommended email"/>
+        <column xsi:type="varchar" name="hash" nullable="false" length="255" comment="UUID hash"/>
```

Comment:
Hash ma być unikalny, więc dobrze by było też wymusić to na poziomie tabeli ustawiając mu unique index

Reply:
Poprawione